cmd/mt-whisper-importer-reader/conversion.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"github.com/kisielk/whisper-go/whisper"
+	"github.com/raintank/metrictank/mdata"
+)
+
+type conversion struct {
+	archives []whisper.ArchiveInfo
+	points   map[int][]whisper.Point
+	method   string
+}
+
+func newConversion(arch []whisper.ArchiveInfo, points map[int][]whisper.Point, method string) *conversion {
+	return &conversion{archives: arch, points: points, method: method}
+}
+
+func (c *conversion) findSmallestLargestArchive(spp, nop uint32) (int, int) {
```

Comment:
do we want to introduce another term spp when we already have span and secondsperpoint?

Reply:
`spp`/`nop` are used pretty much all over that file, because i liked how short they are and that they both have `3` letters. i can change that to `span` though if you want

Reply:
nah, just want to make sure we don't vary our terminology all over the place but this is ok i think